config/app.php

```diff
@@ -93,7 +93,7 @@
 
     'key' => env('APP_KEY', 'SomeRandomString'),
 
-    'cipher' => MCRYPT_RIJNDAEL_128,
+    'cipher' => 'AES-256-CBC',
```

Comment:
Yeah, this changed and we had to update other old Laravel projects accordingly as well.